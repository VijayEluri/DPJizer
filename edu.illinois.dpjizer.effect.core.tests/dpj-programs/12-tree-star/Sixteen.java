/**
 * This file is licensed under the University of Illinois/NCSA Open Source License. See LICENSE.TXT for details.
 */

/**
 * 
 * @author Mohsen Vakilian
 * 
 */
class C<region P> {
	public C() {
	}

	void m() 
	//	reads this:*
	{
		final C<this:[i]>[]<this:[i]>#i zz = null;

		zz[0].m();
	}
}