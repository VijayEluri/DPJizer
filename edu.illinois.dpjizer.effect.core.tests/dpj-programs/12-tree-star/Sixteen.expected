C.C() invokes java.lang.Object.Object()
C.m() reads this : [0] invokes C.m() where { this -> this : [0] : *, P -> this : [0] }
